UnityProject/Assets/Scripts/Shuttles/EscapeShuttle.cs

```diff
@@ -535,7 +536,8 @@ void Action( int time )
 
 	public void SendShuttle()
 	{
-		SoundManager.PlayAtPosition("HyperSpaceBegin", transform.position, gameObject);
+		// JESTER
+		//SoundManager.PlayAtPosition("HyperSpaceBegin", transform.position, gameObject);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unitystation/unitystation/pullRequest?prid=6280689)